eclass/meson.eclass

```diff
@@ -390,7 +396,7 @@ meson_src_compile() {
 		-C "${BUILD_DIR}"
 		--jobs "$(makeopts_jobs "${MAKEOPTS}" 0)"
 		--load-average "$(makeopts_loadavg "${MAKEOPTS}" 0)"
-		--verbose
+		$([[ ${EMESON_VERBOSE} != "OFF" ]] && echo "--verbose")
```

Comment:
I am not the maintainer of this eclass and its up to the maintainers discretion, but personally I would perfer

```
local mesoncompileargs=(
	-C "${BUILD_DIR}"
	--jobs "$(makeopts_jobs "${MAKEOPTS}" 0)"
	--load-average "$(makeopts_loadavg "${MAKEOPTS}" 0)"
)

if [[ ${EMESON_VERBOSE} == ON ]]; then
	mesoncompileargs+=( --verbose )
fi

mesoncompileargs+=( "$@" )
```

Also, I am not a fan of ON / OFF for boolean values, this appears to be more cmake idiomatic. I would use `true` and `false` instead.

Reply:
…and I would use empty/non-empty which is more idiomatic for shell scripts.

Reply:
Oh true that solution does look cleaner. I also find `true` and `false` more logical here; as said the way it is now was inspired by the `cmake.eclass` where in was `ON`/`OFF`.

> I would use empty/non-empty

Do you mean something like `empty (default) -> verbose` and `non-empty -> not verbose`?
I would probably prefer `true (default) -> verbose` and `anything else -> not verbose`.

Reply:
The problem with "anything else" is that if you write `yes` or `on` instead of `true`, things silently go wrong.

Reply:
So you would prefer a variable like `${EMESON_NO_VERBOSE}` which is unset by default and if set to anything disables the `--verbose` flag?

Reply:
`MESON_QUIET`, probably.